src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -137,6 +139,46 @@ public static String extractVersionNumber(String inputString) {
         return inputString.substring(startPos, endPos).replace("-", ".").trim();
     }
     
-    
+    /**
+     * Verifies that the test properties specified in test.properties file allows for HTML
+     * regeneration via God mode to work smoothly (i.e all test HTML files are correctly regenerated, 
+     * strings that need to be replaced with placeholders are correctly replaced, and 
+     * strings that are not supposed to be replaced with placeholders are not replaced).
+     */
+    public void verifyReadyForGodMode() {
+        if (!inst().isDevServer()) {
+            Assumption.fail("God mode regeneration works only in dev server.");
+        }
+        if (!areTestAccountsReadyForGodMode()) {
+            Assumption.fail("Please append a unique id (e.g your name) to each of the default account in"
+                            + "test.properties in order to use God mode, e.g change alice.tmms to "
+                            + "alice.tmms.<yourName>, charlie.tmms to charlie.tmms.<yourName>, etc.");
```

Comment:
This comment is asking the person to append (any) unique id to the default and the person can CONSIDER putting their name. We can set the standard to be appending the name. This means the comment should explicitly ask the person to append his/her name.